src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -392,11 +392,25 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             vkey != VK_SNAPSHOT &&
             keyDown)
         {
+            if (modifiers.IsCtrlPressed() && vkey == 'A')
+            {
+                auto lock = _terminal->LockForWriting();
+                _terminal->SelectAll();
+                _renderer->TriggerSelection();
+                return true;
+            }
+            else if (vkey == VK_RETURN)
+            {
+                CopySelectionToClipboard(false, nullptr);
+                return true;
+            }

```

Comment:
You know what... I should remove this. The user can bind "enter" to "copy" if they want this behavior.